src/main/kotlin/app/utils/RequestException.kt

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.utils
+
+import com.github.kittinunf.fuel.core.FuelError
+
+class RequestException(val exception: Exception)
+    : Exception(exception.message) {
```

Comment:
It's weird it's all aligned with code. shouldn't you do your 8 spaces here? Or why not keep it on the same line?